srcpkgs/quicklisp/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'quicklisp'
+pkgname=quicklisp
+version=20150128
+revision=1
+wrksrc=${pkgname}
+build_style=fetch
+makedepends="sbcl"
+depends="sbcl"
+short_desc="Library manager for Common Lisp"
+maintainer="Rahul <basicfunc@gmail.com>"
+license="custom:bsd"
+homepage="https://beta.quicklisp.org"
+distfiles="${homepage}/quicklisp.lisp"
+checksum=4a7a5c2aebe0716417047854267397e24a44d0cce096127411e9ce9ccfeb2c17
+nocheckperms=yes
```

Comment:
why is this necessary?

Reply:
If you're talking about `nocheckperms`, I added because I was getting error regarding file permission.
Like this

```
quicklisp-20150128_1: post-install_14-fix-permissions: 'find "$PKGDESTDIR" -type f -perm -0002' exited with 1
=> ERROR:   in hook() at common/hooks/post-install/14-fix-permissions.sh:17
=> ERROR:   in run_func() at common/xbps-src/shutils/common.sh:21
=> ERROR:   in run_pkg_hooks() at common/xbps-src/shutils/common.sh:245
=> ERROR:   in main() at common/xbps-src/libexec/xbps-src-doinstall.sh:66
```

Reply:
That's a hook which checks if packages have o=w permissions (anyone on the system can write to this file) which should never happen and can be a security issue. I think this is because the `chmod 777` commands below, why does this package need this?

Reply:
Fixed, it was my mistake